browser/app/js/objects/ObjectsBulkActions.js

```diff
@@ -105,12 +105,12 @@ export class ObjectsBulkActions extends React.Component {
     }
 
     let downloadLabel
-    if (checkedObjects.length === 1 && !checkedObjects[0].endsWith("/")) {
-      downloadLabel = "Download Object"
-    } else if (checkedObjects.length === 1 && checkedObjects[0].endsWith("/")) {
+    if (checkedObjects.length === 1 && checkedObjects[0].endsWith("/")) {
       downloadLabel = "Download as zip"
-    } else {
+    } else if (checkedObjectsCount > 1) {
       downloadLabel = "Download all as zip"
+    } else {
+      downloadLabel = "Download Object"
```

Comment:
After seeing that we have made a change in the order of the download labels, here's the comment I left on the other commit once again:
"Another minor nitpick, however do you think we should make the word Object start with a lowercase letter here (So that this option is Download object instead of Download Object) as all the other options don't have any capital letters after the first letter (Such as Delete selected - s is lowercase, Download as zip, Download all as zip - a and z are lowercase)?"